app/javascript/components/widgets/traffic/panel.jsx

```diff
@@ -1,7 +1,66 @@
 import React from 'react';
+import styled from 'styled-components';
+import { colors, weights, fonts, fontSizes } from '@lib/theme';
+import { GreySubText } from '@components/typography';
 
-const Traffic = () => {
-  return <div>Traffic Panel!</div>;
-};
+const Title = styled.div`
+  margin-top: 16px;
+  margin-bottom: 8px;
+  color: ${colors.white};
+  weight: ${weights.regular};
+  font-size: 24px;
+  font-family: ${fonts.extended};
+`;
+
+const TrafficCam = styled.img`
+  width: 400px;
+  height: 250px;
+`;
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  text-align: center;
+  margin-right: 50px;
+`;
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 700px;
+`;
+
+const Notice = styled(GreySubText)`
+  margin-top: 16px;
+  font-size: ${fontSizes.tiny};
+  text-align: center;
+  margin-right: 750px;
+`;
+
+const Traffic = () => (
+  <>
+    <Row>
+      <Column>
+        <Title>I-95 at Broad St</Title>
+        <TrafficCam src="http://www.dot.ri.gov/img/travel/camimages/95-21 I-95 S @ Broad St .jpg" />
```

Comment:
people who put whitespace in their URLs are savages. 

Would we want to keep these on the backend, since this application wants to be location aware?

Reply:
These should be properly uri encoded. I'm ok with this as an incremental step to putting them in the database.